nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/mongodb/nuxeo.defaults

```diff
@@ -36,3 +36,6 @@ nuxeo.keyvalue.provider=mongodb
 
 ## Make sure we use repository lockmanager
 nuxeo.lock.manager=mongodb
+
+## Use a MongoDB optimized transient store implementation
+# nuxeo.transientstore.provider=mongodb
```

Comment:
Shouldn't we want to keep the KV transientstore for 10.10?

Reply:
yes, this is the reason the line is commented